pybossa/util.py

```diff
@@ -1195,4 +1195,15 @@ def description_from_long_description(desc, long_desc):
         text_desc += "..."
     description = blank_space_regex.sub(" ", text_desc)
     return description if description else " "
-    
\ No newline at end of file
+    
+def get_user_type(firm_num):
+    firm_to_type = current_app.config.get('FIRM_TO_TYPE', {})
+    if firm_num:
+        return firm_to_type.get(int(firm_num), None)
+
+def get_user_data_access_level(firm_num):
+    firm_to_type = current_app.config.get('FIRM_TO_TYPE', {})
+    if firm_num:
```

Comment:
Think can be simplified 
`return ['L2'] if firm_num and int(firm_num) in firm_to_type.keys() else ['L4']`